documentation/macros.sil

```diff
@@ -12,12 +12,13 @@
 \script[src=packages/pdf]
 \script[src=packages/pullquote]
 \script[src=packages/rules]
+\script[src=packages/enumitem-lite]
 \define[command=notehead]{\noindent\bf{\process}\par\medskip}
 \define[command=line]{\novbreak\skip[height=5pt]\noindent\hrule[width=450pt,height=0.3pt]\par\novbreak\skip[height=5pt]}
 \define[command=note]{\medskip
-\set[parameter=document.lskip,value=24pt]
+\set[parameter=document.lskip,value=24pt]{%
```

Comment:
The previous code was changing the setting globally and resetting it to 0 afterwards, I fixed it to have a local scope so that only the note is affected and the surrounding context keeps its lskip, whatever it was. I am not sure the percent is actually necessary, I just wanted to ensure to white space would be inserted here (I tend to do this a lot in my own SILE documents, e.g. with footnotes, lol.) In that very case, at some point (see 4th checkbox in #1194 on "autodoc"), I think this "note" command would anyway have to be refactored to make the package documentation fully independent from the SILE manual macros.

Reply:
I understand the setting scope change, my point was all these `%` escapes should not be necessary. Something is wrong there.

I'm going to let it go however because I've decided the right way to handle this is implement the SILE manual using a new class inheriting from book and adding these macro bits as part of the class init. This *works* but it is cluttered and hard to maintain and not how we are recommend people handle their projects, so we should dogfood a cleaner method, probably with more Lua ;-)

Reply:
I confirm after testing the percent is not necessary here. (Would there be a space in place of it, that would be different, but with just a line break, it works identically). Feel free to remove it if found better. I do hope we'll remove the "note" command at some future point from the manual, so autodoc can provide all necessary tooling for the docs, but that may come later.